File: app/src/main/java/com/radenmas/trendsmarketplace/ui/MainActivity.java
package com.radenmas.trendsmarketplace.ui;

import android.app.Dialog;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.google.android.material.button.MaterialButton;
import com.radenmas.trendsmarketplace.R;
import com.radenmas.trendsmarketplace.databinding.ActivityMainBinding;

/**
 * Created by RadenMas on 28/03/2022.
 */
public class MainActivity extends AppCompatActivity {

    private ActivityMainBinding b;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        b = ActivityMainBinding.inflate(getLayoutInflater());
        View v = b.getRoot();
        getWindow().getDecorView().setSystemUiVisibility(View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR);
        setContentView(v);

        onClick();
    }

    private void onClick() {
        b.imgAbout.setOnClickListener(view -> {
            startActivity(new Intent(this, AboutActivity.class));
        });
        b.imgLogout.setOnClickListener(view -> {
            dialogLogout();
        });
        b.rlRisetData.setOnClickListener(view -> {
            startActivity(new Intent(this, RisetDataActivity.class));
        });
        b.rlTokopedia.setOnClickListener(view -> {
            startActivity(new Intent(this, TokopediaActivity.class));
        });
        b.rlShopee.setOnClickListener(view -> {
            startActivity(new Intent(this, ShopeeActivity.class));
        });
        b.rlBlibli.setOnClickListener(view -> {
            startActivity(new Intent(this, BlibliActivity.class));
        });
        b.rlGuide.setOnClickListener(view -> {
            startActivity(new Intent(this, GuideActivity.class));
        });
    }

    private void dialogLogout() {
        Dialog dialog = new Dialog(this, R.style.DialogStyle);
        dialog.setContentView(R.layout.dialog_logout);
        dialog.getWindow().setBackgroundDrawableResource(R.drawable.bg_dialog);

        MaterialButton btnYes = dialog.findViewById(R.id.btnYes);
        MaterialButton btnNo = dialog.findViewById(R.id.btnNo);

        btnYes.setOnClickListener(view1 -> {
            finish();
        });
        btnNo.setOnClickListener(view1 -> {
            dialog.dismiss();
        });

        dialog.show();
    }
}

File: app/src/main/java/com/radenmas/trendsmarketplace/ui/MainActivity.kt
package com.radenmas.trendsmarketplace.ui

import android.app.Dialog
import android.content.Intent
import android.os.Build
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.google.android.material.button.MaterialButton
import com.radenmas.trendsmarketplace.R
import com.radenmas.trendsmarketplace.databinding.ActivityMainBinding

/**
 * Created by RadenMas on 28/03/2022.
 */
class MainActivity : AppCompatActivity() {

    private lateinit var b: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        b = ActivityMainBinding.inflate(layoutInflater)
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
            window.decorView.systemUiVisibility = View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR
        }
        setContentView(b.root)

        initView()
        onClick()
    }

    private fun onClick() {
        // fungsi untuk membuka info aplikasi
        b.imgAbout.setOnClickListener {
            startActivity(Intent(this, AboutActivity::class.java))
        }

        // fungsi untuk tombol logout
        b.imgLogout.setOnClickListener {
            dialogLogout()
        }

        // fungsi untuk membuka menu riset data
        b.rlRisetData.setOnClickListener {
            startActivity(Intent(this, RisetDataActivity::class.java))
        }

        // fungsi untuk membuka menu tokopedia
        b.rlTokopedia.setOnClickListener {
            startActivity(Intent(this, TokopediaActivity::class.java))
        }

        // fungsi untuk membuka menu shopee
        b.rlShopee.setOnClickListener {
            startActivity(Intent(this, ShopeeActivity::class.java))
        }

        // fungsi untuk membuka menu blibli
        b.rlBlibli.setOnClickListener {
            startActivity(Intent(this, BlibliActivity::class.java))
        }

        // fungsi untuk membuka menu panduan aplikasi
        b.rlGuide.setOnClickListener {
            startActivity(
                Intent(this, GuideActivity::class.java)
            )
        }
    }

    private fun initView() {

    }

    private fun dialogLogout() {
        // tampilan popup ketika akan keluar
        val dialog = Dialog(this, R.style.DialogStyle)
        dialog.setContentView(R.layout.dialog_logout)
        dialog.window!!.setBackgroundDrawableResource(R.drawable.bg_dialog)
        val btnYes = dialog.findViewById<MaterialButton>(R.id.btnYes)
        val btnNo = dialog.findViewById<MaterialButton>(R.id.btnNo)
        btnYes.setOnClickListener {
            finish()
        }
        btnNo.setOnClickListener {
            dialog.dismiss()
        }
        dialog.show()
    }
}